events/wc_companion_events.txt

```diff
@@ -0,0 +1,1407 @@
+﻿namespace = companion
```

Comment:
From the first glance, I see no need to script so many events with just 3 different lines in each.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121093948-dc811600-c7fe-11eb-9c2c-e456be1f3096.png)

</details>

Reply:
Removed all but one